kratos/containers/model.cpp

```diff
@@ -72,7 +72,7 @@ namespace Kratos
         else
         {
             if(&(*search->second) != &*(pModelPart))
-                KRATOS_ERROR << "trying to add to the Model two DISTINCT model parts with the same name. This should be possible (and it will be in the future) if they belong to two different root model_parts, but it is currently disallowed";
+                KRATOS_WARNING("Model") << "trying to add to the Model two DISTINCT model parts with the same name. This should be possible (and it will be in the future) if they belong to two different root model_parts, but it is currently disallowed" << std::endl;
```

Comment:
:+1:

Reply:
@RiccardoRossi after thinking abt this some more, what happens if a modelpart with the same name is used, will the existing one be overwritten?

Reply:
the answer is ...NOTHING ...meaning that if a modelpart with the same name is already in model nothing new is added, that is, the old one remains untouched and the new one is not added

Reply:
Is there a special reason this can't stay an error until then?

Reply:
in any case i just brought this back to the master version...